staging/src/k8s.io/legacy-cloud-providers/azure/azure_wrap.go

```diff
@@ -277,7 +277,13 @@ func (az *Cloud) newNSGCache() (*timedCache, error) {
 	getter := func(key string) (interface{}, error) {
 		ctx, cancel := getContextWithCancel()
 		defer cancel()
-		nsg, err := az.SecurityGroupsClient.Get(ctx, az.ResourceGroup, key, "")
+		var rg string
```

Comment:
could we assign `az.SecurityGroupResourceGroup` to `az.ResourceGroup` if it is not set in `InitializeCloudFromConfig()`? So that we would use `az.SecurityGroupResourceGroup` here.

Reply:
Done